powerbi-docs/developer/visuals/capabilities.md

```diff
@@ -12,12 +12,13 @@ ms.date: 06/18/2019
 
 # Capabilities and properties of Power BI visuals 
 
-You use capabilities to provide information to the host about your visual. All properties on the capabilities model are `optional`.
+You use capabilities to provide information to the host about your visual. All properties on the capabilities model are `optional` except `privileges`, which are mandatory.
```

Comment:
- [ ] ```
[]()
```suggestion
You use capabilities to provide information to the host about your visual. All properties on the capabilities model are `optional` except `privileges`, - [ ] which are mandator

```
```

```